js/data.js

```diff
@@ -4,7 +4,7 @@ var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
-  EntryId: 1
+  entryId: 1
```

Comment:
This should still be called `nextEntryId`, you are only changing the name for the individual entry objects you create